src/components/UserCard.tsx

```diff
@@ -4,31 +4,68 @@ import { UserCardType } from "../types/UserCard";
 
 const UserCard: React.FC<UserCardType> = (props: UserCardType) => {
   const { name, profile } = props;
-  const { description, school, year, skills, commitmentLevel } = profile;
+  const { description, school, year, major, skills, commitmentLevel } = profile;
+
+  // Custom style for the school section if it's "Georgia Institute of Technology"
+  const schoolStyle =
+    school === "Georgia Institute of Technology"
+      ? {
+          color: "gold",
+          fontWeight: "bold",
+        }
+      : {};
+
+  // Custom style for the commitment tag based on the commitment level
+  const commitmentStyle = 
+  commitmentLevel === "low"
+    ? {
+        bg: "red.400",
+        color: "white",
+      }
+    : commitmentLevel === "medium"
+    ? {
+        bg: "yellow.400",
+        color: "black",
+      }
+    : commitmentLevel === "high"
+    ? {
+        bg: "green.400",
+        color: "white",
+      }
+    : {};
+
 
   return (
     <Box
       borderWidth="1px"
       borderStyle="solid"
       rounded="lg"
       boxShadow="lg"
-      height="400px"
-      width="400px"
+      height="350px"
+      width="350px"
       fontWeight="bold"
       alignItems="center"
       backgroundColor="white"
       padding="4"
+      marginBottom="20px"
     >
       <Flex flexDirection="column">
         <Text fontSize="3xl" fontWeight="bold" mb="2">
           {name}
         </Text>
+        <Text fontSize="sm" mb="2" style={schoolStyle}>
+          {school}
+        </Text>
+        <Text fontSize="sm" mb="2">
+          {year}
+        </Text>
         <Text fontSize="sm" mb="2">
-          {`${school}, ${year}`}
+          {major}
```

Comment:
These changes have become outdated with some of the recent changes made. You'll have to resolve them when merging main to your branch. Let's not show major in the user card since it might look like a lot of info, and instead show it in the user modal that we have implemented. So don't worry about adding that now, we may  have it later.